modal/_utils/function_utils.py

```diff
@@ -296,12 +289,13 @@ def class_parameter_info(self) -> api_pb2.ClassParameterInfo:
         signature = _get_class_constructor_signature(self.user_cls)
         for param in signature.parameters.values():
             has_default = param.default is not param.empty
-            if param.annotation not in CLASS_PARAM_TYPE_MAP:
-                raise InvalidError("modal.parameter() currently only support str or int types")
-            param_type, default_field = CLASS_PARAM_TYPE_MAP[param.annotation]
-            class_param_spec = api_pb2.ClassParameterSpec(name=param.name, has_default=has_default, type=param_type)
+            if param.annotation not in PYTHON_TO_PROTO_TYPE:
+                raise InvalidError("modal.parameter() currently only support str, int, or bytes types")
```

Comment:
nit
```suggestion
                raise InvalidError("modal.parameter() currently only supports str, int, or bytes types")
```

Reply:
Also might be nice to build the error message programmatically, feels like we could easily forget to expand it in the future if we add more type support.

Reply:
Turns out this error message is never normally triggered, since there was a very similar validation check that runs as part of the Cls.from_local constructor (which does have programmatic generation of the message). I refactored the two to use the same common validator now